components/automate-ui/src/app/entities/jobs/job.effects.ts

```diff
@@ -58,7 +58,7 @@ export class JobEffects {
     ofType(JobActionTypes.JOB_UPDATE_SUCCESS),
     map(() => new CreateNotification({
       type: Type.info,
-      message: 'Updated a scan job.'
+      message: 'Updated scan job.' // TODO: Add the scan job ID here
```

Comment:
Author note: consistency -- every such message needs to include the ID of the object.